src/test/mocks/@interlay/interbtc-api/parachain/vaults.ts

```diff
@@ -10,6 +10,11 @@ const DEFAULT_VAULT_WITH_ISSUABLE_TOKENS = (_accountId: AccountId, collateralCur
 });
 
 const mockVaultsGet = jest.fn((accountId, currency) => DEFAULT_VAULT_WITH_ISSUABLE_TOKENS(accountId, currency));
+// ray test touch <
+// const vault1Id = newVaultId(mockInterBtcApi.api as ApiPromise, '5GQoBrhX3mfnmKnw2qz2vGvHG8yvf6xT15gGM54865g6qEfE', RELAY_CHAIN_NATIVE_TOKEN, KBtc);
+// const vaultsWithIssuableTokens = new Map().set(vault1Id, new BitcoinAmount(10));
+// const mockVaultsGetVaultsWithIssuableTokens = jest.fn(() => vaultsWithIssuableTokens);
 const mockVaultsGetVaultsWithIssuableTokens = jest.fn(() => []);
```

Comment:
You need to return array of vaults here in order to be able to create issue request - if the array is empty it means there are not vaults, so it's not possible to create issue request.